protos/messaging.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "protos/common.proto";
+
+option go_package = "github.com/nivista/steady/internal/.gen/protos/messaging";
+
+message Key {
+    string domain = 1;
+    string timerUUID = 2;
+}
+
+message Create {
+    Task task = 1;
+    Schedule schedule = 2;
+    Meta meta = 3;
+}
+
+message Execute {
+    Progress progress = 1;
+    bytes result = 2;
```

Comment:
result is simply a blob intended for elasticsearch. It should be JSON. This is in protos/messaging.proto, putting it in protos/internal/messaging.proto does nothing. However, the generated code is internal.